<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadSingleVideo</class>
 <widget class="QMainWindow" name="DownloadSingleVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elrouby Youtube Downloader: Download Single Video</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: #D6F4ED
}

QPushButton
{
	background-color: #473472;
	border-radius: 0;
	color: white
}

QLineEdit, QTextEdit, QComboBox
{
	background-color: #87BAC3;
	color: white;
	border-radius: 0
}

QProgressBar::chunk
{
	background-color: #87BAC3
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="videourl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Video URL</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="playerlayout"/>
   </widget>
   <widget class="QLineEdit" name="title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>561</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="desc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="comments">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>631</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #53629E;
color: white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadbtn">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletebtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pastebtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>11</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="Quality">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality</string>
    </property>
   </widget>
   <widget class="QTableView" name="singlevideoshowtable">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>71</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>⏸️</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>220</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
